"hit show script clawpack" fails

```
~/repos/hashstack2(master)$ hit show script clawpack
[ERROR] Uncaught exception:
Traceback (most recent call last):
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/cli/main.py", line 169, in help_on_exceptions
    return func(*args, **kw)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/cli/main.py", line 143, in main
    retcode = args.subcommand_handler(ctx, args)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/cli/frontend_cli.py", line 37, in run
    self = cls(ctx, args)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/cli/frontend_cli.py", line 33, in __init__
    self.builder = ProfileBuilder(self.ctx.logger, self.source_cache, self.build_store, self.profile)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/spec/builder.py", line 28, in __init__
    self._compute_specs()
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/spec/builder.py", line 86, in _compute_specs
    traverse_depth_first(pkgname)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/spec/builder.py", line 83, in traverse_depth_first
    process(pkgname, pkgspec)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/spec/builder.py", line 72, in process
    if self.build_store.is_present(self._build_specs[pkgname]):
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/core/build_store.py", line 348, in is_present
    return self.resolve(build_spec.artifact_id) is not None
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/core/build_store.py", line 329, in resolve
    self._log_artifact_collision(path)
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/core/build_store.py", line 271, in _log_artifact_collision
    """ % dict(path=path))
  File "/projects/b1cc51a6-c4a2-4281-98b4-a5c2b1dea21b/repos/hashdist/hashdist/hdist_logging.py", line 154, in log_lines
    for line in textwrap.wrap(textwrap.dedent(text), 70).splitlines():
AttributeError: 'list' object has no attribute 'splitlines'

This exception has not been translated to a human-friendly error message,
please file an issue at https://github.com/hashdist/hashdist/issues pasting
this stack trace.
```

--- a/hashdist/hdist_logging.py
+++ b/hashdist/hdist_logging.py
@@ -151,7 +151,7 @@ class Logger(object):
             self.set_error_occurred(True)
 
     def log_lines(self, level, text):
-        for line in textwrap.wrap(textwrap.dedent(text), 70).splitlines():
+        for line in textwrap.wrap(textwrap.dedent(text), 70):
             self.log(level, line)
 
     def debug(self, msg, *args):

